contrib/golang/filters/http/source/cgo.cc

```diff
@@ -200,6 +200,19 @@ CAPIStatus envoyGoFilterHttpSendPanicReply(void* r, void* details) {
   });
 }
 
+CAPIStatus envoyGoFilterHttpSetStringFilterState(void* r, void* key, void* value, int state_type,
+                                                 int life_span, int stream_sharing) {
+  return envoyGoFilterHandlerWrapper(r,
+                                     [key, value, state_type, life_span, stream_sharing](
+                                         std::shared_ptr<Filter>& filter) -> CAPIStatus {
+                                       auto key_str = copyGoString(key);
```

Comment:
seems referGoString could be better, since `filterState()->setData(key,` accept the `string_view` type `key`, we could save a memory copy here.

and we still need to copy the `key` when it not in the safe thread, before dispatcher post.

Reply:
done, thanks.